src/web/lib/components/ThemeBackgroundPicker/index.js

```diff
@@ -46,6 +47,21 @@ class ThemeBackgroundPicker extends React.Component {
     this.setState({ selected: false });
   }
 
+  handleKeyPress(event) {
+    if (event.keyCode === 27) {
```

Comment:
It's probably worth creating a `const` at the top of the file for the key code so we can use a more descriptive variable name.

For example, `const ESC = 27;`, so this can turn into `if (event.keyCode === ESC)`, or whatever.